vm/src/byteslike.rs

```diff
@@ -124,3 +125,26 @@ impl<'a> BorrowValue<'a> for PyRwBytesLike {
         self.0.as_contiguous_mut().unwrap()
     }
 }
+
+pub enum BufOrStr {
```

Comment:
is there a concept for this type in CPython?

Reply:
Yeah, it's like the `s*` format code for PyArg_Parse: https://docs.python.org/3/c-api/arg.html#strings-and-buffers

I could add a doc comment for that.

Reply:
got it. it named `Py_buffer` but we use it for different struct.